pmm_ps_setup/start-pmm-ps.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#Fetch PMM-QA Repo, this contains pmm-framework script, playbook required for the tests
+#$PS_VERSION Variable will be fetched from the github actions env variable.
+
+echo $PS_VERSION
+
+PS_TARBALL_PATH=https://downloads.percona.com/downloads/TESTING/ps-$PS_VERSION/Percona-Server-$PS_VERSION-Linux.x86_64.glibc2.28.tar.gz
+
+# Delete if the Repo already checkedout
+sudo rm -r pmm-qa || true 
+
+git clone -b PMM-7-tarball-ms-pxc https://github.com/percona/pmm-qa/
```

Comment:
this should have been merged as I know, if now lets use an env variable for branch name to test some changes in future